fabtests/Makefile.am

```diff
@@ -401,6 +402,11 @@ benchmarks_fi_rdm_pingpong_SOURCES = \
 	$(benchmarks_srcs)
 benchmarks_fi_rdm_pingpong_LDADD = libfabtests.la
 
+benchmarks_fi_rma_pingpong_SOURCES = \
```

Comment:
Can you also add this to the Windows build? It goes [here](https://github.com/ofiwg/libfabric/blob/main/fabtests/fabtests.vcxproj)